text/sigma-protocol/README.md

```diff
@@ -0,0 +1,76 @@
+- Feature Name: proof-of-knowledge-of-discrete-logarithms
+- Start Date: 2019-08-03
+- RFC PR: (leave this empty)
+- Ursa Issue: (leave this empty)
+- Version: 0.1
+
+# Summary
+[summary]: #summary
+
+This is a design document for constructing proofs of knowledge of committed values using the Sigma protocol. The prover is able to construct both interactive and non-interactive proofs of knowledge.
+
+# Motivation
+[motivation]: #motivation
+
+Several protocols require proving knowledge of discrete logarithms. Like proving knowledge of messages and randomness in a Pedersen commitment, a vector Pedersen commitment. Such protocols then become part of larger protocols like proving knowledge of a signature or selectively revealing some of the messages in the commitment. These protocols form the basis for anonymous credentials in Hyperledger Aries.
```

Comment:
The title says "proof of knowledge of discrete logarithms". The summary says "proof of knowledge of committed values". This sentence now switches back to "proving knowledge of discrete logarithms". It might be obvious to a cryptographer that, since modular arithmetic is going to be used, discrete logarithms are the natural way to prove knowledge of committed values. But this is not obvious to me as a less mathematical person. Perhaps your middle sentence could say "proofs of knowledge of committed values, by demonstrating the mathematical relationship between the known value and its discrete logarithm" -- or something like that? Isn't the ability to show you know the discrete logarithm of X a means to an end, and the end is to demonstrate that you know X?